lighthouse-core/gather/gatherers/start-url.js

```diff
@@ -37,9 +37,15 @@ class StartUrl extends Gatherer {
     }
 
     try {
-      return await this._attemptStartURLFetch(passContext.driver, startUrlInfo.startUrl);
+      const statusAndExplanation =
+        await this._attemptStartURLFetch(passContext.driver, startUrlInfo.startUrl);
+      return {url: startUrlInfo.startUrl, ...statusAndExplanation};
     } catch (err) {
-      return {statusCode: -1, explanation: 'Error while fetching start_url via service worker.'};
+      return {
+        url: startUrlInfo.startUrl,
+        statusCode: -1,
+        explanation: 'Error while fetching start_url via service worker.',
```

Comment:
this whole file needs some i18n love @exterkamp

Reply:
I think all our gatherers have this problem :)